content/blog/svglite-2-0-0/index.Rmd

```diff
@@ -0,0 +1,250 @@
+---
+output: hugodown::hugo_document
+
+slug: svglite-2-0-0
+title: svglite 2.0.0
+date: 2021-02-08
+author: Thomas Lin Pedersen
+description: >
+    svglite 2.0.0 is now on CRAN. This is a huge feature release and part rewrite, and should excite anyone who cares about vector graphics. Read on to find out what is new!
+
+photo:
+  url: https://unsplash.com/photos/pVoEPpLw818
+  author: Rodion Kutsaev
+
+# one of: "deep-dive", "learn", "package", "programming", or "other"
+categories: [package] 
+tags: [graphic-device, svglite]
+---
+
+```{=html}
+<!--
+TODO:
+* [x] Pick category and tags (see existing with `post_tags()`)
+* [x] Find photo & update yaml metadata
+* [x] Create `thumbnail-sq.jpg`; height and width should be equal
+* [x] Create `thumbnail-wd.jpg`; width should be >5x height
+* [x] `hugodown::tidy_thumbnail()`
+* [ ] Add intro sentence
+* [ ] `use_tidy_thanks()`
+-->
+```
+We're extremely happy to announce the release of [svglite](https://svglite.r-lib.org) 2.0.0.
+svglite is a graphic device that is capable of creating SVG files from R graphics.
+SVG is a vector graphic format which means that it encode the instructions for recreating a graphic in a scale-independent way.
+This is in contrast with raster graphics such as PNG (as can be produced with the graphic devices in [ragg](https://ragg.r-lib.org)) which encodes actual pixel values and will get pixelated as you zoom in.
+
+You can install the latest release of svglite from CRAN with:
+
+```{r, eval = FALSE}
+install.packages("svglite")
+```
+
+Much time has passed since svglite had a major release and this blog post will go into detail with all the major changes.
+You can see a full list of changes in the [release notes](%7B%20github_release%20%7D).
+
+## Motivation for svglite
+
+As part of this release we have sharpened our personal motivation for creating and maintaining svglite in the face of the already existing `svg()` device provided by R.
+All of the changes that are part of this release somehow plays into these motivations and are thus grouped by it below.
+
+### Speed
```

Comment:
I think you should put the biggest benefit up front: so do editability, then speed, then file size.